src/api/polls.js

```diff
@@ -28,6 +34,9 @@ export async function patchPoll(pollId, poll) {
     },
     body: JSON.stringify(poll)
   });
+  if (!response.ok) {
+    throw new Error(response.statusText);
+  }
   const patchedPoll = await response.json();
   return patchedPoll;
```

Comment:
I'm not sure if you need this code (maybe for debugging?). However, currently your  `patchPoll` function returns a `patchedPoll`. When called on the page `Vote-poll.js`, the returned `patchedPoll` is not used on that page.